### DISCUSSION TITLE
Is ort compatible with WASM

### DISCUSSION THREAD

**Discussion by DawChihLiou** [pykeio/ort#32] [Q&A] [answered]
Hey everyone,

Thanks for your amazing work. I'd like to use the crate to create embedding transformers in WASM. I found ort because I tried many crates but I haven't been able make it work. I'd like to give it a shot so I'd like to see if anyone already has experience with ort with WASM. Any tip, story, suggestion are appreciated. Thanks so much!

**Comment by decahedron1** [comment, marked as answer]
ICYMI (and sorry to necro), `ort` has supported WASM since v1.15.5. Prebuilt binaries provided too!

**Reply by DawChihLiou** [reply]
Awesome! I'm excited to try it.
